src/ROUTES.js

```diff
@@ -81,7 +81,7 @@ export default {
     getReportDetailsRoute: reportID => `r/${reportID}/details`,
     REPORT_SETTINGS: 'r/:reportID/settings',
     getReportSettingsRoute: reportID => `r/${reportID}/settings`,
-    LOGIN_WITH_SHORT_LIVED_TOKEN: 'transition',
+    TRANSITION: 'transition',
```

Comment:
Could we please name this something else? I keep seeing it and wondering "what transition is this talking about?" What does "transition" mean and what does it apply to? When would I know to use or reference this route?

Reply:
Ok, how about "transitionFromOldDot"?